diff -ur openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f.orig/CMakeLists.txt openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f/CMakeLists.txt
--- openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f.orig/CMakeLists.txt	2024-11-03 14:01:44.000000000 +0100
+++ openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f/CMakeLists.txt	2024-11-05 22:30:51.409115362 +0100
@@ -794,7 +794,6 @@
   src/ext/libtess2/Source/sweep.c
   src/ext/libtess2/Source/tess.c
   src/ext/lodepng/lodepng.cpp
-  src/ext/polyclipping/clipper.cpp
   src/geometry/ClipperUtils.cc
   src/geometry/Geometry.cc
   src/geometry/GeometryCache.cc
@@ -1177,10 +1176,8 @@
     src/gui/parameter/ParameterWidget.ui
 )
 
-if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-  # Ignore specific warning on external lib
-  set_source_files_properties("src/ext/polyclipping/clipper.cpp" PROPERTIES COMPILE_FLAGS "-Wno-class-memaccess")
-endif()
+pkg_check_modules(polyclipping REQUIRED polyclipping)
+target_link_libraries(OpenSCAD PRIVATE ${polyclipping_LIBRARIES})
 
 if (ENABLE_MANIFOLD)
   target_compile_definitions(OpenSCAD PRIVATE ENABLE_MANIFOLD)
Only in openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f: CMakeLists.txt~
diff -ur openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f.orig/src/core/OffsetNode.h openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f/src/core/OffsetNode.h
--- openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f.orig/src/core/OffsetNode.h	2024-11-03 14:01:44.000000000 +0100
+++ openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f/src/core/OffsetNode.h	2024-11-05 22:26:57.833523910 +0100
@@ -1,7 +1,7 @@
 #pragma once
 
 #include "core/node.h"
-#include "polyclipping/clipper.hpp"
+#include <polyclipping/clipper.hpp>
 
 #include <string>
 
Only in openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f/src/core: OffsetNode.h~
diff -ur openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f.orig/src/geometry/ClipperUtils.h openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f/src/geometry/ClipperUtils.h
--- openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f.orig/src/geometry/ClipperUtils.h	2024-11-03 14:01:44.000000000 +0100
+++ openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f/src/geometry/ClipperUtils.h	2024-11-05 22:27:27.884243264 +0100
@@ -1,6 +1,6 @@
 #pragma once
 
-#include "polyclipping/clipper.hpp"
+#include <polyclipping/clipper.hpp>
 #include "geometry/Polygon2d.h"
 
 #include <memory>
Only in openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f/src/geometry: ClipperUtils.h~
diff -ur openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f.orig/src/libsvg/shape.h openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f/src/libsvg/shape.h
--- openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f.orig/src/libsvg/shape.h	2024-11-03 14:01:44.000000000 +0100
+++ openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f/src/libsvg/shape.h	2024-11-05 22:27:49.318089688 +0100
@@ -38,7 +38,7 @@
 
 #include <boost/optional.hpp>
 
-#include "polyclipping/clipper.hpp"
+#include <polyclipping/clipper.hpp>
 
 namespace libsvg {
 class shape;
Only in openscad-fd3a9aad2bcd913ac1830e11670f0a422231e43f/src/libsvg: shape.h~
