diff --git a/openscad.pro b/openscad.pro
index a2eefe6..20b1864 100644
--- a/openscad.pro
+++ b/openscad.pro
@@ -440,8 +440,7 @@ SOURCES += src/version_check.cc \
 HEADERS += src/ext/lodepng/lodepng.h
            
 # ClipperLib
-SOURCES += src/ext/polyclipping/clipper.cpp
-HEADERS += src/ext/polyclipping/clipper.hpp
+LIBS += -lpolyclipping
 
 # libtess2
 INCLUDEPATH += src/ext/libtess2/Include
diff --git a/src/clipper-utils.h b/src/clipper-utils.h
index 0dc7cd0..c98cda0 100644
--- a/src/clipper-utils.h
+++ b/src/clipper-utils.h
@@ -1,6 +1,6 @@
 #pragma once
 
-#include "ext/polyclipping/clipper.hpp"
+#include <polyclipping/clipper.hpp>
 #include "Polygon2d.h"
 
 namespace ClipperUtils {
--- openscad-openscad-2019.01-RC4/src/libsvg/shape.h~	2019-03-17 02:57:25.000000000 +0100
+++ openscad-openscad-2019.01-RC4/src/libsvg/shape.h	2019-04-19 14:37:29.511178262 +0200
@@ -34,7 +34,7 @@
 #include <Eigen/Geometry>
 
 #include "util.h"
-#include "ext/polyclipping/clipper.hpp"
+#include <polyclipping/clipper.hpp>
 
 namespace libsvg {
 
--- openscad-2021.01/CMakeLists.txt.orig	2021-01-31 21:17:41.000000000 +0100
+++ openscad-2021.01/CMakeLists.txt	2022-06-07 21:45:07.303640010 +0200
@@ -403,6 +403,9 @@
   endif()
 endif()
 
+pkg_check_modules(polyclipping REQUIRED polyclipping)
+list(APPEND COMMON_LIBRARIES ${polyclipping_LIBRARIES})
+
 include_directories("src/ext/libtess2/Include")
 
 # NOTE: To keep HEADLESS builds working, do NOT add Qt-dependent sources here,
@@ -495,7 +497,6 @@
   src/UserModule.cc
   src/Tree.cc
   src/ext/lodepng/lodepng.cpp
-  src/ext/polyclipping/clipper.cpp
   src/ext/libtess2/Source/bucketalloc.c
   src/ext/libtess2/Source/dict.c
   src/ext/libtess2/Source/geom.c
